src/Microsoft.DotNet.Compiler.Common/Executable.cs

```diff
@@ -109,8 +112,11 @@ private void CopyAssets(IEnumerable<LibraryExport> libraryExports)
         private void WriteDepsFileAndCopyProjectDependencies(LibraryExporter exporter)
         {
             WriteDeps(exporter);
-            WriteRuntimeConfig(exporter);
-            WriteDevRuntimeConfig(exporter);
+            if (!_context.ProjectFile.HasRuntimeOutput(_configuration))
+            {
+                WriteRuntimeConfig(exporter);
+                WriteDevRuntimeConfig(exporter);
```

Comment:
@schellap Please, review this. This breaks our initial assumption that you would always have a .dev.config file. Is this an ok change to make?